### PR TITLE
Update Sql capabilities API to include elastic pool edition capabilities

### PR DESCRIPTION
**Pull request by nathannfan**
Elastic Pool edition capabilities were recently lit up in canary clusters, and will soon be in production.  Also, temporarily removing APIs that are not yet public in production from compositeSql.json.

### PR information
- [ x] The title of the PR is clear and informative.
- [x ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x ] If applicable, the PR references the bug/issue that it fixes.
- [x ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x ] My spec meets the review criteria:
  - [x ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by nathannfan** [comment]
SDK Pull Request: https://github.com/Azure/azure-sdk-for-net/pull/3078

File: arm-sql/compositeSql.json [hunk 1]
```diff
@@ -13,8 +13,6 @@
     "./2014-04-01/swagger/servers.json",
     "./2014-04-01/swagger/sql.core.json",
     "./2014-04-01/swagger/databaseSecurityAlertPolicies.json",
-    "./2015-05-01-preview/swagger/blobAuditingPolicies.json",
```
Comment (dsgouda) [inline arm-sql/compositeSql.json]: Are you merging these into a single swagger? If not, are ```failoverGroups.json``` or ```virtualNetworkRules.json``` going to be used anywhere?
@vishrutshah will this affect the Ruby builds?

**Review by dsgouda — CHANGES_REQUESTED** [review]
Just have a question, other than that looks very good!

File: arm-sql/compositeSql.json (hunk 1 above)
Comment (nathannfan) [inline arm-sql/compositeSql.json]: So these APIs aren't yet live in production, so I want to remove them from the compositeSql.json file for now until we're ready to actually use the client they'll generate.  Is that okay?

Comment (dsgouda) [inline arm-sql/compositeSql.json]: In that case, consider deleting the unused files and revert whenever you need them.

Comment (nathannfan) [inline arm-sql/compositeSql.json]: Is that a blocker?  It's a reasonable thing to do, but I didn't check in these files, and they both already went through the Pull request review process, so I'd hate for anyone to have to review already-reviewed files.  

If there's a policy against having swagger files not referenced by the composite file, we may have an issue later on, as we have two versions of some APIs and I will be adding the new version and replacing the reference to the old version in the composite file, but not deleting the old file.

Comment (dsgouda) [inline arm-sql/compositeSql.json]: Had offline chat with reviewers and we think this should be okay. I need to point out however, that these would be breaking changes.

**Review by dsgouda — APPROVED** [review]

**Pull request merged**